app/channels/discussion_post_list_channel.rb

```diff
@@ -0,0 +1,14 @@
+class DiscussionPostListChannel < ApplicationCable::Channel
+  def self.notify!(discussion, iteration)
+    broadcast_to("#{discussion.id}_#{iteration.id}", {})
+  end
+
+  def subscribed
+    discussion = Solution::MentorDiscussion.find(params[:discussion_id])
```

Comment:
This ensures that only the student and the mentor can subscribe to the channel.

Reply:
```suggestion
    discussion = Solution::MentorDiscussion.find_by!(uuid: params[:discussion_id])
```